src/app/components/list/overview/index.html

```diff
@@ -60,6 +60,120 @@
+      propertyName="pageNumber"
+      isOptional="true"
+    >
+    Specifies the current page number. This property accepts <sky-code>number</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="pageSize"
+      isOptional="true"
+    >
+    Specifies the number of items to display per page. This property accepts <sky-code>number</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="search"
+      isOptional="true"
+    >
+    Specifies a function to dynamically manage the data source when users change the text in the list field. The search function must return an array or a promise of an array. The <sky-code>search</sky-code> property is particularly useful when the data source does not live in the source code. Acceptable value: <sky-code>(searchText: string, data?: any[]) => any[] | Promise&lt;any[]&gt;</sky-code>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="sort"
+      isOptional="true"
+    >
+    Specifies a set of fields to sort by. Acceptable value: <sky-code>Array of ListSortModel</sky-code> properties.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListFilterModel properties">
+    <sky-demo-page-property
+      propertyName="name"
+      isOptional="true"
+    >
+    Specifies the name of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="label"
+      isOptional="true"
+    >
+    Specifies the text for the filter's summary item. If not specified, then the filter summary item text defaults to the <sky-code>value</sky-code> of the filter. This property accepts <sky-code>string</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="dismissable"
+      isOptional="true"
+    >
+    Indicates whether users can dismiss the filter's summary item. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="value"
+      isOptional="true"
+    >
+    Specifies the current value of the filter. This property accepts <sky-code>any</sky-code> value.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="defaultValue"
+      isOptional="true"
+    >
+    Specifies the default value of the filter. If the value of the filter is set to the default value, then the filter is not applied.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="filterFunction"
+      isOptional="true"
+    >
+    Specifies the function to apply to determine whether an item is filtered. This property is required when using in memory data provider. Acceptable values: <sky-code>(item: ListItemModel, filter: any) => boolean</sky-code>. This property accepts <sky-code>boolean</sky-code> values.    
```

Comment:
Except for the last sentence, this description is the same as the description of the `filters` property of `ListDataRequestModel`. Is that correct? I'm pretty sure we don't want the last sentence here. And looking at the code, I'm not sure if the Acceptable values sentence in the `filters` description is there. It looks like the last sentence there should be "This property accepts a `ListFilterModel` value" but you might want to check with a dev.

If it's correct that the description here should be the same as for `filters`, I'd recommend the same changes here that I suggested there.

Reply:
> It looks like the last sentence there should be "This property accepts a ListFilterModel value" but you might want to check with a dev.

Yes an no. The last sentence should be this:

> This property accepts a function of type `(item: ListItemModel, filter: any) => boolean`